frontend/src/features/videos/PublicStatusAction.tsx

```diff
@@ -40,9 +40,11 @@ export const UserRatingPublicToggle = ({
     <Box display="flex" alignItems="center" flexWrap="wrap" width="100%" px={1}>
       <Typography variant="caption" style={{ fontSize: '11px' }}>
         {nComparisons > 0 ? (
-          <Trans t={t} i18nKey="video.nComparisonsByYou" count={nComparisons}>
-            {{ count: nComparisons }} comparison by you
-          </Trans>
+          <a href={`/comparisons/?video=${videoId}`}>
+            <Trans t={t} i18nKey="video.nComparisonsByYou" count={nComparisons}>
+              {{ count: nComparisons }} comparison by you
+            </Trans>
+          </a>
```

Comment:
![Screenshot 2022-01-30 at 22 02 24](https://user-images.githubusercontent.com/8818081/151717700-d7c5c99f-7145-48ed-83e7-b62bcabd663e.png)

Please avoid the blue & purple styles for the text. 

The best would be to have a similar style as for the channel name and possibly a similar Tooltip which may say "See my comparisons".

![Screenshot 2022-01-30 at 22 03 00](https://user-images.githubusercontent.com/8818081/151717757-65404ee9-492e-4040-b439-20c65ce77517.png)

Reply:
The implemention of this link is visible here:
https://github.com/tournesol-app/tournesol/blob/1fcb04e2f889e4a9204bdf98bdf2e505acb17b4a/frontend/src/features/videos/VideoCard.tsx#L286-L294

`<Link>` is provided by MUI to get consistent styling.
`<RouterLink>` is another component provided by react-router to navigate internally in the single-page app (whereas clicking on`<a>` would trigger an unnecessary page reload).